### PR TITLE
Swiping upgrade

### PR DESCRIPTION
**Pull request by Ellord207**

### CODE REVIEW

File: src/frontend-samples/swiping-viewport/SwipingComparisonApi.ts [hunk 1]
```diff
@@ -177,7 +163,7 @@ abstract class SampleTiledGraphicsProvider implements TiledGraphicsProvider {
     viewport.view.forEachTileTreeRef(func);
 
     // Do not apply the view's clip to this provider's graphics - it applies its own (opposite) clip to its graphics.
-    this.viewFlagOverrides.showClipVolume = false;
+    this.viewFlagOverrides.showClipVolume = true;
```
Comment (pmconne) [inline src/frontend-samples/swiping-viewport/SwipingComparisonApi.ts:166]: Why did this change from `false` to `true`?

Comment (Ellord207) [reply src/frontend-samples/swiping-viewport/SwipingComparisonApi.ts:166]: That was an accident.  I do remember it didn't have an effect, as it's redundancy from when the showcase shared a single viewport.

Comment (pmconne) [reply src/frontend-samples/swiping-viewport/SwipingComparisonApi.ts:166]: I don't follow. From your email description, the swiping sample still divides a single viewport using a clip plane and its inverse.
If you revert this to `false`, rebuild, and run, the same behavior is observed?

> 7.	The TiledGraphicsProvider.addToScene method will
> a.	***Replace the viewports clip with one created using the negated normal (flipping it).***
> b.	Make the reality data transparent
> c.	Create a new Scene context based on the viewport
> d.	Create a new graphic branch and add the graphics created by the scene to it.
> e.	Create graphics from the branch by calling the render system
> f.	Add the graphics to the viewport’s scene.
> g.	***Reset the old clipping plane to the viewport.***
> h.	Make the reality data opaque.

Comment (Ellord207) [reply src/frontend-samples/swiping-viewport/SwipingComparisonApi.ts]: I revented it back to false, and still obvser the same bahavor.

I also pushed the revert.

**Pull request merged**